data/json/items/ammo.json

```diff
@@ -563,7 +563,7 @@
     "category": "chems",
     "price": 500,
     "price_postapoc": 2000,
-    "name": { "str": "mixed smokeless gunpowder" },
+    "name": { "str_sp": "mixed smokeless gunpowder" },
```

Comment:
Part of me wants to make the plural "units of smokeless gunpowder," but I don't know if "units" would be too...ambiguous?

Reply:
From the "count" below, it's multiple units in the first place, it looks like.

Reply:
Well, it may not be multiple units, but I'm unsure exactly what a "unit" would be... for powders, really, it should be number of full containers, number of pounds/kg, or number of L/whatever.